<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_tab_Widget</class>
 <widget class="QTabWidget" name="Main_tab_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <property name="movable">
   <bool>true</bool>
  </property>
  <property name="tabBarAutoHide">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>CMT Parser</string>
   </attribute>
   <widget class="QProgressBar" name="parse_progress_Bar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="result_filename_textBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>car</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="Parse_result_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CMT Parser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CMT_parse_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Uruchom parser CMT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj nazwę pliku, do którego zapisane zostaną wyniki parsowania</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CMT_directory_textbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CMT_all</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj nazwę katalogu, w którym znajdują się pliki CMT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Parsuje CMT z katalogu i zapisuje graf do pliku.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj ścieżkę</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CMT_project_path_textbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>d:\\projekty\\zajęcia na uczelni\\!tpd\\</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit_a_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zamknij</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Graph Analyser</string>
   </attribute>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Graph Analyser</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Analizuje plik wskazany na zakładce CMT Parser</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz kierunek analizy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_subgraph_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wygeneruj i zapisz podgraf</string>
    </property>
   </widget>
   <widget class="QListWidget" name="object_type_list">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz rodzaj obiektu do analizy</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="object_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>To tylko taka porada.</string>
    </property>
    <property name="statusTip">
     <string>To jakiś status.</string>
    </property>
    <property name="whatsThis">
     <string>Pole do wyboru obiektu do analizy.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz obiekt z grafu do analizy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_lineedit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj fragment nazwy</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="range_list">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="subgraph_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit_b">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zamknij</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Table Ranking Tool</string>
   </attribute>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Table Ranking Tool</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Wylicza ranking dla nowej kwerendy</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="wylicz_ranking_pb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wylicz ranking</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wpisz nazwę obiektu</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="nazwa_TRT_lineedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="wyniki_TRT_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>391</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="openLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lub zaczytaj obiekt wybrany na zakładce Graph Analyser</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="zaczytaj_obiekt_pb">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zaczytaj obiekt</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Wybierz rodzaj obiektu</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="Gb_rb">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Group by</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="From_rb">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Where_rb">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Where</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Del_From_pb">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear_From_pb">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear_Where_pb">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Del_Where_pb">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear_Gb_pb">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Del_Gb_pb">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lista_obiektow_From">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Lista_obiektow_Where">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Lista_obiektow_Gb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit_c">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zamknij</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit_c</sender>
   <signal>clicked()</signal>
   <receiver>Main_tab_Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExit_a_2</sender>
   <signal>clicked()</signal>
   <receiver>Main_tab_Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExit_b</sender>
   <signal>clicked()</signal>
   <receiver>Main_tab_Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
